### PR TITLE
enterprise update and a little content

### PR DESCRIPTION
**Pull request by ZNicoZ**

### CODE REVIEW

**Comment by svettwer** [comment]
Hi!
I briefly checked the PR. The alignment within the enterprise boxes is broken now. Is this something you want @tnobody to fix?

File: data/enterprise/price_table.toml [hunk 1]
```diff
@@ -24,7 +29,9 @@ priceLink="https://www.github.com/sakuli/sakuli"
 [[plans]]
 code="M"
 instances="< 4"
-sla="24h"
+maintenance=true
+sla="2 business days"
```
Comment (svettwer) [inline data/enterprise/price_table.toml]:
```suggestion
sla="Two business days"
```

File: data/enterprise/price_table.toml [hunk 2]
```diff
@@ -46,7 +55,9 @@ priceLink="https://www.consol.de/anfrage-sakuli/"
 [[plans]]
 code="XL"
 instances="∞"
-sla="3h"
+maintenance=true
+sla="4h"
```
Comment (svettwer) [inline data/enterprise/price_table.toml]:
```suggestion
sla="4 hours"
```

File: themes/sakuli/layouts/enterprise/list.html [hunk 3]
```diff
@@ -28,10 +28,16 @@ <h2>{{ .code }}</h2>
...
+            {{ if .maintenance }}
+            <li>
+                Maintenance<a href="#maintenance" class="info"></a>
+            </li>
+            {{end}}
             {{ if isset . "sla"}}
             <li class="meta-info">
                 <span>
-                    <strong>{{.sla}}</strong> SLA Time
+                    Professional Support<br />initial reaction time<br />
```
Comment (svettwer) [inline themes/sakuli/layouts/enterprise/list.html]: I think this is too much text. IMHO it looks cluttery.

Comment (svettwer) [inline themes/sakuli/layouts/enterprise/list.html]: I think this is too much text. IMHO it looks cluttery.
Maybe _support reaction time_?

File: themes/sakuli/layouts/enterprise/list.html [hunk 4]
```diff
@@ -70,11 +81,24 @@ <h2>{{ .code }}</h2>
...
 
+{{if isset $priceList "MaintenanceDescription"}}
+<dl class="price-list-description shrink-content">
+    <dt>
+        <a id="maintenance">
+            <h2>What does <i>Maintenance</i> mean?</h2>
+        </a>
+    </dt>
+    <dd>
+        {{$priceList.SlaDescription}}
```
Comment (svettwer) [inline themes/sakuli/layouts/enterprise/list.html]:
```suggestion
        {{$priceList.MaintenanceDescription}}
```

Comment (tnobody) [reply themes/sakuli/layouts/enterprise/list.html]: @ZNicoZ please review this changes regarding to content changes

**Comment by ZNicoZ** [comment]
> Hi!
> I briefly checked the PR. The alignment within the enterprise boxes is broken now. Is this something you want @tnobody to fix?

Yes, talked about it with @tnobody - he will fix it.

**Pull request merged**